controllers/imagerepository_controller.go

```diff
@@ -90,7 +151,12 @@ func (r *ImageRepositoryReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return ctrl.Result{}, err
 	}
 
+	componentIdForMetrics := fmt.Sprintf("%s=%s", imageRepository.Labels[ComponentNameLabelName], imageRepository.Namespace)
```

Comment:
ImageRepository doesn't have to have `ComponentNameLabelName` label. For example, JBS uses it without the label.
I think we have to have identifier not related to Component, but to instance of ImageRepository itself.

Reply:
will use name + namespace then